style/styles.css

```diff
@@ -0,0 +1,1101 @@
+  width: 1100px;
+}
+
+.personOfWeek h2 {
+  font-size: 25px;
+  margin-bottom: 30px;
+}
+
+.linkTexts {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.linkTexts h2 {
+  color: #666;
+  margin-bottom: 20px;
+}
+
+.linkTexts h1 a:hover {
+  color: #d33b2b;
+}
+
+.shippingNow #element5 a {
+  background: #fff;
+  color: #277dcc;
+  padding: 30px;
+  margin-right: 25px;
+  margin-left: 25px;
+  font-size: 20px;
+  border-radius: 12px;
+}
+
+.communityLinks {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  color: black;
+  margin-top: 70px;
+}
+
+.linkContainer {
+  display: flex;
+  justify-content: start;
+  flex-direction: row;
+  margin-left: auto;
+  margin-right: auto;
+  width: 700px;
+  height: 300px;
+}
+
+@media (max-width: 800px) {
+  .linkContainer {
+    display: flex;
+    justify-content: start;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-left: auto;
+    margin-right: auto;
+    width: auto;
+    height: auto;
+  }
+
+  .communityLinks {
+    display: grid;
+    grid-template-columns: 1fr;
+    color: black;
+    margin-top: 70px;
+  }
+}
+
+.linkContainer .imgContainer {
+  margin: auto;
+  margin-top: 0;
+  padding-right: 30px;
+}
+
+.linkContainer .imgContainer .profileImg {
+  border: 5px solid #d33b2b;
+}
+
+.dateAndComment {
+  display: flex;
+  flex-direction: row;
+}
+
+.wholeCommunityLinks {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.more-content-button {
+  font-family: "Cormorant Garamond", serif;
+  color: #333;
+  text-decoration: none;
+  border-bottom: #d33b2c 0.225em solid;
+}
+
+.seeAllLatestArticles {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 100px;
+}
+
+.more-content-button:hover {
+  font-family: "Cormorant Garamond", serif;
+  color: #d33b2c;
+}
+
+.wholeFeaturedBook {
+  margin-top: 60px;
+  background-color: #d33b2b;
+  color: white;
+}
+
+.featuredBook {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap-reverse;
+  height: 400px;
+  color: white;
+  width: 86%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.featuredBookText {
+  margin-top: 400px;
+  flex: 70%;
+  padding: 40px;
+}
+
+.featuredBookImage {
+  width: 20%;
+  height: 600px;
+  flex: 30%;
+  transform: rotate(-15deg);
+  transform-origin: left bottom;
+}
+
+.featuredBookImage img {
+  transform: scale(0.6);
+}
+
+.featuredBookImage:hover {
+  transform: rotate(0deg);
+}
+
+.featuredBookText h5 {
+  font-family: "Cormorant Garamond", serif;
+  font-size: 20px;
+  margin-bottom: 40px;
+}
+
+.featuredBookText h3 {
+  font-size: 20px;
+  margin-bottom: 30px;
+}
+
+.tableOfContentsButton {
+  background-color: white;
+  border: none;
+  color: #277ccc;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 12px;
+  font-size: 100%;
+  text-shadow: #1f2c43;
```

Comment:
You have only added color to the `text-shadow`. But it needs two more values -> `h-shadow` and ` v-shadow`. 
https://www.w3schools.com/cssref/css3_pr_text-shadow.asp

Reply:
Changes implemented.